app/components/application_record_card/view.html.erb

```diff
@@ -1,22 +1,24 @@
 <div class="app-application-card">
-  <div class="app-application-card_col">
-    <h<%= heading_level %> class="govuk-heading-m govuk-!-margin-bottom-3">
-      <%= govuk_link_to trainee_name, view_trainee(record) %>
-    </h<%= heading_level %>>
-    <%= trainee_id %>
-    <%= trn %>
-  </div>
+  <div class="app-application-col-trainee-course">
+    <div class="app-application-col-trainee">
+      <h<%= heading_level %> class="govuk-heading-m govuk-!-margin-bottom-3 app-application-trainee_name">
```

Comment:
We can probably remove this margin override which I think your're overriding above anyway.